R/initialize_modules.R

```diff
@@ -627,12 +631,36 @@ initialize_fims <- function(parameters, data) {
       dplyr::pull(type) |>
       unique()
 
+    # Initialize age composition module if the data type includes "age" and
+    # if "AgeComp" exists in the data distribution specification
+    if ("age" %in% fleet_types &
+        "AgeComp" %in% names(parameters[["modules"]][["fleets"]][[fleet_names[i]]][["data_distribution"]])) {
```

Comment:
this line and other if statements that use `names(parameters[["modules"]][["fleets"]][[fleet_names[i]]][["data_distribution"]])` would benefit from the creation of an object rather than repeating this long code, e.g.,
```
distribution_names_for_fleet_i <- names(
  parameters[["modules"]][["fleets"]][[fleet_names[i]]][["data_distribution"]]
)
...
if ("age %in% fleet_types & "AgeComp" %in% distribution_names_for_fleet_i) {
```
Also, I am pretty sure that the `&` should be `&&` on line 636

Reply:
Done